correios/exceptions.py

```diff
@@ -54,34 +54,18 @@ class InvalidShippingLabelError(ModelError):
 
 
 class InvalidPackageError(ModelError):
-    pass
-
-
-class InvalidPackageDimensionsError(InvalidPackageError):
-    pass
-
 
-class InvalidMinPackageDimensionsError(InvalidPackageDimensionsError):
-    pass
-
-
-class InvalidMaxPackageDimensionsError(InvalidPackageDimensionsError):
-    pass
+    def __init__(self, errors):
+        self.errors = errors
 
-
-class InvalidPackageWeightError(InvalidPackageError):
-    pass
-
-
-class InvalidMinPackageWeightError(InvalidPackageWeightError):
-    pass
-
-
-class InvalidMaxPackageWeightError(InvalidPackageWeightError):
-    pass
+    def __str__(self):
+        if len(self.errors) > 1:
+            return "Invalid Package: {} and {}".format(", ".join(self.errors[:-1]), self.errors[-1])
+        else:
+            return "Invalid Package: {}".format(self.errors[0])
 
 
-class InvalidPackageSequenceError(InvalidPackageError):
+class InvalidPackageSequenceError(ModelError):
```

Comment:
humm... I'd like to keep a common ancestor for "Invalid Package" in this case.

Reply:
I can use change `InvalidPackageError` to a base exception again. But,  I'll need another one for dimensions and weight errors. Do you have any ideas for a name?